app/src/main/java/org/simple/clinic/registration/facility/RegistrationFacilitySelectionScreen.kt

```diff
@@ -64,7 +67,8 @@ class RegistrationFacilitySelectionScreen(context: Context, attrs: AttributeSet)
             searchQueryChanges(),
             retryClicks(),
             facilityClicks(),
-            locationPermissionChanges()
+            locationPermissionChanges(),
+            activityResultStream()
         ),
         controller = controller,
         screenDestroys = RxView.detaches(this).map { ScreenDestroyed() }
```

Comment:
We can extract `RxView.detaches(this).map { ScreenDestroyed() }` as a separate variable here and then use that both while setting up the events and pass it as a parameter to `activityResultStream`.

Reply:
I have done that in latter commits when migrating to rxbinding 3